recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: 73df4d5d34f0468bd57d09f2d8af363e95ed6cc3a4a86129d2f2c366259902a2
 
 build:
-    number: 0
+    number: 1
```

Comment:
No need to bump the build number. There is no functional change. (Actually no change at all until `conda-forge` moves to `conda-build 2` and we can un-comment that test.)